e2e/experimental/accordion/app.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+import shiny.experimental as x
```

Comment:
Because this file is here, please delete file: `shiny/experimental/e2e/accordion/app.py`.